src/components/ImageTemplate.tsx

```diff
@@ -35,6 +35,7 @@ const ImageTemplate: React.FC<ImageTemplateProps> = ({
           top: '10px',
           right: '10px',
           overflow: 'hidden',
+          zIndex: -1,
```

Comment:
This was actually an oversight in the original implementation, though only a factor on some smaller screens.